nix/pkgs/kubernetes-tools.nix

```diff
@@ -3,7 +3,7 @@
 
 buildGoModule rec {
   pname = "kubernetes";
-  version = "1.23.16";
+  version = "1.28.2";
```

Comment:
I'm a bit surprised that the `hash` didn't change. Have you checked if this intented change became active?

On my machine there's still the old version in the nix env:

```
➜  wire-server-deploy git:(bump_kubernetes_to_1_28) kubectl version
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.16", GitCommit:"60e5135f758b6e43d0523b3277e8d34b4ab3801f", GitTreeState:"archive", BuildDate:"1980-01-01T00:00:00Z", GoVersion:"go1.21.4", Compiler:"gc", Platform:"linux/amd64"}
```

Reply:
nice find.. I need to update the hash